src/app/shared/listing-card/listing-card.component.scss

```diff
@@ -0,0 +1,118 @@
+.listing-card {
+    max-width: 100%;
+    position: relative;
+    height: auto;
+    border: none;
+    border-radius: 0.625rem;
+
+    .hero-img {
+        width: 100%;
+        border-radius: 0.625rem;
+        transition: .3s ease;
+    }
+
+    .sale_type {
+        position: absolute;
+        top: 5%;
+        left: 3%;
+        display: flex;
+        align-items: center;
+        gap: 0.625rem;
+
+        .btn {
+            background-color: #1A1A1A;
+            color: #fff;
+            padding: 0.4rem 0.75rem;
+            border: none;
+            border-radius: 3.125rem;
+            cursor: pointer;
+            font-weight: 600;
+            font-size: 0.65rem;
+            transition: .3s ease-in;
+            text-transform: uppercase;
+  
+            &:hover {
+                background-color: #fff;
+                color: #1A1A1A;
+                border: 2px solid #1A1A1A;
```

Comment:
This color is defined at serval places. declare is color in your global style sheet and use it throughout the application

Reply:
done